sigopt/xgboost/run.py

```diff
@@ -1,4 +1,5 @@
 import xgboost
+# pylint: disable=no-name-in-module
```

Comment:
you can remove this after making the suggested change to `setup.py`

Reply:
It seems pylint works at the `sigopt` directory, when import xgboost, it will add `sigopt/xgboost` as xgboost module, so  even I have installed xgboost, the lint check still does not pass.